bower.json

```diff
@@ -7,16 +7,15 @@
     "output"
   ],
   "dependencies": {
-    "purescript-typelevel": "^3.0.0",
-    "purescript-maybe": "^3.0.0",
-    "purescript-functors": "^2.2.0",
-    "purescript-uint": "^0.5.0",
-    "purescript-eff": "^3.1.0",
-    "purescript-functions": "^3.0.0"
+    "purescript-typelevel": "https://github.com/whoadave/purescript-typelevel.git#master",
```

Comment:
I've temporarily set bower to point to my working fork and created this PR so that you can review, but this PR shouldn't be merged until the purescript-typelevel PR has been merged and this line updated